pageheader/pageheader.php

```diff
@@ -33,7 +35,7 @@ function pageheader_uninstall() {
 
 
 
-function pageheader_addon_settings(&$a,&$s) {
+function pageheader_addon_settings(App $a, $s) {
```

Comment:
The `&` operator of the `$s` parameter enables this function to change the hook data. If you remove it this function has no effect whatsoever.